doc/nrf/releases/release-notes-1.7.0.rst

```diff
@@ -0,0 +1,601 @@
+  * The DFU socket API is now deprecated.
+
+NFC
+---
+
+* Updated the NFCT interrupt configuration to reduce the ISR latency and enable scheduling decision in the interrupt context.
+* Updated the :ref:`nfc_uri` library to allow encoding of URI strings longer than 255 characters.
+
+Trusted Firmware-M
+------------------
+
+* Added a test case for the secure read service that verifies that only addresses within the accepted range can be read.
+* Updated :file:`tfm_platform_system.c` to fix a bug that returned ``TFM_PLATFORM_ERR_SUCCESS`` instead of ``TFM_PLATFORM_ERR_INVALID_PARAM`` when the address passed is outside of the accepted read range.
+
+Common Application Framework (CAF)
+----------------------------------
+
+* Added :ref:`caf_net_state`.
+* Added :ref:`caf_power_manager`.
+* Updated :ref:`caf_sensor_sampler` with a limit to the number of ``sensor_event`` events that it submits.
+
+Profiler
+--------
+
+* Added profiling string data.
+* Optimized numeric data encoding.
+
+Edge Impulse
+------------
+
+* Added support for Thingy:53 to :ref:`nrf_machine_learning_app`.
+* Added configuration for nRF52840 DK that supports data forwarder over NUS to :ref:`nrf_machine_learning_app`.
+
+Pelion
+------
+
+* Updated Pelion Device Management Client library version to 4.10.0.
+* Switched to using :ref:`caf_power_manager` and :ref:`caf_net_state` in :ref:`pelion_client`.
+* Updated the :ref:`application documentation <pelion_client>` with a step that requires downloading Pelion development tools.
+
+Build system
+------------
+
+* Bugfixes:
+
+  * Fixed a bug where :file:`dfu_application.zip` would not be updated after rebuilding the code with changes.
+
+MCUboot
+=======
+
+The MCUboot fork in |NCS| (``sdk-mcuboot``) contains all commits from the upstream MCUboot repository up to and including ``7a51968``, plus some |NCS| specific additions.
+
+The code for integrating MCUboot into |NCS| is located in :file:`ncs/nrf/modules/mcuboot`.
+
+The following list summarizes the most important changes inherited from upstream MCUboot:
+
+* Added AES support for image encryption (based on mbedTLS).
+* MCUboot serial: Ported encoding to use the cddl-gen module (which removes dependency on the `TinyCBOR`_ library).
+* bootutil_public library: Made ``boot_read_swap_state()`` declaration public.
+
+Mcumgr
+======
+
+The mcumgr library contains all commits from the upstream mcumgr repository up to and including snapshot ``e64f5060b``.
+
+The following list summarizes the most important changes inherited from upstream mcumgr:
+
+* Fixed an issue with SMP file management commands that would fail to read or write files, or both.
+
+Zephyr
+======
+
+.. NOTE TO MAINTAINERS: All the Zephyr commits in the below git commands must be handled specially after each upmerge and each NCS release.
+
+The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``14f09a3b00``, plus some |NCS| specific additions.
```

Comment:
@mbolivar-nordic , can you confirm this is the correct commit ID? One last check before merge.